cmd/krel/cmd/ff.go

```diff
@@ -139,9 +142,12 @@ func runFf(opts *ffOptions) error {
 
 	prepushMessage(repo.Dir(), kgit.DefaultRemote, branch, opts.org, releaseRev, headRev)
 
-	_, pushUpstream, err := util.Ask("Are you ready to push the local branch fast-forward changes upstream? Please only answer after you have validated the changes.", "yes", 3)
-	if err != nil {
-		return err
+	pushUpstream := true
```

Comment:
should be `false` as default value? to never push to upstream?

Reply:
Yeah I kinda turned the logic around, should be now more clear :+1: